src/Symfony/Component/Cache/Psr16Cache.php

```diff
@@ -11,6 +11,7 @@
 
 namespace Symfony\Component\Cache;
 
+use Composer\InstalledVersions;
```

Comment:
Let's use 3.0+ as version number and drop using this class

Reply:
Sounds good! Fixed and force-pushed :)